classification/templates/classification/tags/condition.html

```diff
@@ -2,7 +2,10 @@
 {% if condition.terms %}
     {% for term in condition.terms %}
         {% if not forloop.first %}<br/>{% endif %}
-        <span><a href="{% url 'ontology_term' term.url_safe_id %}" class="hover-link">{{ term.id }}</a> {{ term.name }}</span>
+        {% if no_links %}
+            <span> {{ term.id }} {{ term.name }}</span>
+        {% else %}
+            <span><a href="{% url 'ontology_term' term.url_safe_id %}" class="hover-link">{{ term.id }}</a> {{ term.name }}</span>
+        {% endif %}
     {% endfor %}
-    {% if condition.join_text %}&#32;<span class="font-italic">{{ condition.join_text }}</span>{% endif %}
```

Comment:
If we change ClassificationGroup as per the above suggestion, then we can put this back in.

Especially since {% condition %} is used in other contexts where we would want to show off the join text